codes/zig/chapter_stack_and_queue/array_stack.zig

```diff
@@ -90,7 +90,7 @@ pub fn main() !void {
     std.debug.print("\n栈的长度 size = {}", .{size});
 
     // 判断栈是否为空
-    var is_empty = stack.empty();
+    var is_empty = stack.is_empty();
```

Comment:
isEmpty()

Reply:
I found `is_empty()` in array_stack.zig but `empty()` in linkedlist_stack.zig

Please unify the function naming.

Reply:
I found is_empty() in array_stack.zig but empty() in linkedlist_stack.zig

Please unify the function naming.

Reply:
done.